_python/pgbench_python.py

```diff
@@ -45,14 +45,22 @@ def psycopg_connect(args):
                             port=args.pgport)
     return conn
 
+def psycopg2_connect(args):
```

Comment:
Wrong module used above on line 44, should be `psycopg.connect`

Reply:
thanks for that.